server/src/common/enums/error-messages.enum.ts

```diff
@@ -3,4 +3,6 @@ export enum ErrorMessagesEnum {
 
   FEATURES_NOT_FOUND = 'FEATURES_NOT_FOUND',
   FEATURE_NOT_FOUND = 'FEATURE_NOT_FOUND',
+  GENRES_NOT_FOUND = 'GENRES_NOT_FOUND',
```

Comment:
Our convention is to add empty line between errors of different modules
```suggestion

  GENRES_NOT_FOUND = 'GENRES_NOT_FOUND',
```